### PR TITLE
Update Readme for V1.3

### LINKED ISSUE #9: Update Readme to reflect latest ID changes

### PR DESCRIPTION
**Pull request by aivantg**
## What's in this PR
Edited Readme!! 

Included some assumptions but I'm probably missing some if you want to suggest any. 
Please help me make sure the language makes sense!

### Related Issues
closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,62 @@
...
-5. Add convenient run script
+The `output` folder is where you'd like your generated files to live. 
+The mode parameter accepts "auto", "auto-headless", and "manual"
+
+**Other Parameters**: 
+`input`: Only required if mode is set to Manual. Similar to `output`, specifies where `schemaRaw.json` can be found. Details below
+`defaultView`: You can specify a default view to use for Read requests. Default is "Grid View"
+`schemaMeta`: Where your metadata for schema generation lives. Details below
+
+specifying where your `schemaMetadata.json` file lives as the input folder and where you'd like to store your utility functions as the output folder.
```
Comment (annieyro) [inline README.md]: Nitpick: discovered while testing locally that `schemaMeta.json` assumes an object is passed in, not a file anymore, right?

File: README.md [hunk 2]
```diff
@@ -1,43 +1,62 @@
...
+
+#### 3b) If using auto or auto-headless mode, create a `.airtable.env` file.
+
+Create a file called `.airtable.env` (identical to `.env.example`) in the root of your project and fill in the values with your airtable account email and password. This information is only saved on your computer in this hidden file.
```
Comment (annieyro) [inline README.md]: nitpick, but if we're going to call `.airtable.env`, any thoughts on having the example file be called `.example.env` (or vice versa) ?

File: README.md [hunk 3]
```diff
@@ -80,13 +121,38 @@ export const getAnnouncementsByProjectGroup = async value => {
 ```
 This is in addition to the two default "get" functions created. 
 
-## Screenshots
+## Record Transformations
```
Comment (annieyro) [inline README.md:171]: After #19 is closed, will update with the pluralize + sort + filter functionality too

File: README.md [hunk 4]
```diff
@@ -47,13 +66,35 @@ Update your scripts object in `package.json` to include the following
...
 ```
 
+Optional, add ` && pretty-quick` or your own prettifier command to the script to post-process the schema code. 
+
 ## Running the script
 
 Run `npm run generate-schema` to re-generate your utility functions every time your airtable base updates!
 
-## Optional Inputs
+## Important Assumptions
```
Comment (annieyro) [inline README.md:122]: Does this still assume the Airtable API key is stored as `process.env.REACT_APP_AIRTABLE_API_KEY`? I have to change that line every time haha

**Review by annieyro — APPROVED** [review]
Nice!! Added some nitpicks, but very cool updates (-:

File: README.md (hunk 1 above)
Comment (aivantg) [reply README.md]: oh yes good catch i thought i had updated it

File: README.md (hunk 3 above)
Comment (aivantg) [reply README.md:171]: Sounds good! I was imagining that all future PRs update the PR

File: README.md [hunk 5]
```diff
@@ -80,13 +130,43 @@ export const getAnnouncementsByProjectGroup = async value => {
...
+3. Split on spaces
+4. Capitalize first character of each word except the first
+5. Combine
+
+#### 2. Accurate Linked Record Column Names
 
-Airtable Schema Downloading Code Credit: https://github.com/cape-io/airtable-schema
+Linked Records on Airtable are usually named something like "Author" or "Project", which would make the corresponding javascript-y name "author" or "project". The most expressive name, however, given how they come back in the API response, would be "authorId" or "projectId". 
+
+Record transformation accounts this, pluralizing it when the record is marked as a one-to-many relationship
```
Comment (annieyro) [inline README.md:194]: Nit: 
"accounts for this, pluralizing `Id` when the linked record is marked as a one-to-many relationship

File: README.md [hunk 6]
```diff
@@ -80,13 +130,43 @@ export const getAnnouncementsByProjectGroup = async value => {
 ```
 This is in addition to the two default "get" functions created. 
 
-## Screenshots
+## Record Transformations
```
Comment (annieyro) [inline README.md:171]: Nit: do you want to mention the pluralizing of table names here as well?
"Table names are pluralized for the `getAllRecords` and `getRecordsByAttribute` generated functions in `request.js` using a naive check of whether the table name ends in an `s`."

File: README.md [hunk 7]
```diff
@@ -1,43 +1,71 @@
 # airtable-schema-generator
 
-An NPM package with a script to download an Airtable schema, and generate schema file, constants, and request helpers! 
+An NPM package designed to support React + Node apps that use Airtable as a backend!
```
Comment (annieyro) [inline README.md:3]: Add a Table of Contents! https://ecotrust-canada.github.io/markdown-toc/

File: README.md [hunk 8]
```diff
@@ -1,43 +1,71 @@
...
+The helper files establish abstraction barriers for the rest of your project to follow. At the lowest level is **`airtable.js`**, which contains the functions that directly interact with airtable. Next, comes **`request.js`**, which are CRUD functions generated for _every_ table in your Airtable base. Finally is **`schema.js`**, a source of truth for your airtable schema.
 
-Details on the design decisions behind the architecture can be found [here](https://www.notion.so/calblueprint/PP-Power-Airtable-Client-Side-6a1b6734af294ef88609a6d6d256ca3d).
+Besides organizing your code's interactions with airtable via helper functions, the package also makes Airtable records easier to work with! The file `schema.js` contains mappings from Airtable Column Names to a more javascript-y version of them. For example, "First Name" becomes "firstName". It's aware of linked records as well; "Project Group" (which might be a one-to-one linked record column) will become "projectGroupId". 
+
+In your codebase, you can use the javascript-y names, and the provided `airtable.js` file will automatically transform them to and from the actual Airtable column names!
```
Comment (annieyro) [inline README.md]: Personally I found it very helpful to look through `schema.js` and `request.js` to sanity-check what functions I was calling, so maybe could add a line here that they can?

And/or link to the Records Transformation section!

**Review by annieyro — CHANGES_REQUESTED** [review]
Nitpicks!

**Pull request merged**